tests/e2e/Services/Databases/Legacy/DatabasesCustomServerTest.php

```
@@ -5243,4 +5243,936 @@ public function testBulkDelete(): void
+            'x-appwrite-key' => $this->getProject()['apiKey']
+        ]), [
+            'key' => 'string',
+            'size' => 128,
+            'required' => false,
+        ]);
+
+        sleep(1);
+
+        $createDate = '2000-01-01T10:00:00.000+00:00';
+        $updateDate = '2000-02-01T15:30:00.000+00:00';
+        $date1 = '2000-01-01T10:00:00.000+00:00';
```

PHPMD (2.15.0)

Avoid unused local variables such as '$date1'. (Unused Code Rules)

(UnusedLocalVariable)